similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/RestrictedPatientSimilarityView.java

```diff
@@ -130,19 +127,6 @@ public DocumentReference getReporter()
         }
     }
 
-    @Override
-    protected PatientPhenotypeSimilarityView createPhenotypeSimilarityView(Set<? extends Feature> match,
-        Set<? extends Feature> reference, AccessType access)
-    {
-        return new RestrictedPatientPhenotypeSimilarityView(match, reference, access);
-    }
-
-    @Override
-    protected DisorderSimilarityView createDisorderSimilarityView(Disorder match, Disorder reference, AccessType access)
-    {
-        return new RestrictedDisorderSimilarityView(match, reference, this.access);
-    }
-
     @Override
```

Comment:
if the only thing restricted is genes, do we need `RestrictedPatientSimilarityView` and not just `(restricted)GenotypeSimilarityView`?